source/index.html.md

```diff
@@ -1824,7 +1827,7 @@ Field Name | Type | Description | Units
 **side** | String | The order side (buy, sell) | -
 **status** | String | The order's status (queued, open, partial-fill, closed) | -
 **type** | String | The order type (market, limit) | -
-**client_id** | String | The client_id if any | -
+**origin_id** | String | The origin_id if any | -
```

Comment:
What is the plan to change the API endpoints that have `client_id` in their name?

Reply:
Will create the tasks to change them, and follow the query param for doing the same.

Reply:
Shouldn't you remove them from the docs now if you want to change them? And Notify any users?

Reply:
They should be removed from the documentation but we can keep them for those users that are already using it, a metric can be added to identify which users keep using them and then we can contact them so w can remove the method at all.